guide/getting-started/hardware.md

```diff
@@ -54,16 +54,18 @@ General purpose hardware, like personal computers and smartphones, work great fo
    height = 400
 %}
 
-Many users back up their wallet using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from physical destruction from fire and water. [Seedplate](https://seedplate.com/) is an example of such a product.
+Many users back up their wallet using metal plates that have their recovery phrase etched into them. This protects the recovery phrase from [various forms](https://blog.lopp.net/metal-bitcoin-seed-storage-stress-test-iv/) of physical destruction. [Seedplate](https://seedplate.com/) is an example of such a product.
 
-A simpler option is writing your recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
+A simpler option is writing a recovery phrase on a piece of paper. Paper can be easily destroyed compared to steel, making this option less secure.
 
 Some applications, such as [Muun](https://blog.muun.com/why-not-just-a-mnemonic/) have users print out encrypted PDF backups that contain all the required recovery data, not just the recovery phrase.
 
 See [bitcoin backups]({{ '/guide/private-key-management/backups/' | relative_url }}) for more details.
 
 ## Hardware wallets
 
+_Also known as external signers_
```

Comment:
This looks weird as a stand-alone sentence on the preview page. Adding a period might help.

Reply:
Without punctuation, it functions like a subheading. Adding a period would turn it into an incomplete sentence, so we shouldn't do that, @GBKS. 

@Bosch-0 I'd recommend doing one of these:

- Leaving as is
- Merging it with one of the sentences, e.g. "Bitcoin hardware wallets, also known as _signers_, act like bitcoin-centric security keys."
- Add it to a [tip box](https://bitcoin.design/guide/contribute/formatting/#tips).

Reply:
I'll go with point 2